frontend/packages/integration-tests-cypress/views/timestamp.ts

```diff
@@ -0,0 +1,3 @@
+export const timestamp = {
```

Comment:
Hi, do you feel there will be many other helper methods for timestamp?  Not sure it needs to be in it's own file, perhaps move to `views/common.ts`

Reply:
Hi @rebeccaalpert, thanks for the cypress updates, LGTM!  Last little nit. ^^

Reply:
All set @dtaylor113! Thanks.